CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 2.8.12)
+cmake_minimum_required(VERSION 3.0.0)
```

Comment:
Why exactly is this necessary?

Reply:
I no longer have a warning when switching to version 3.0.0, And it's the same version as for the SDL2 lib: https://github.com/libsdl-org/SDL/blob/main/CMakeLists.txt#L5

Reply:
> I no longer have a warning when switching

You never mentioned any warnings: If you post the warnings you get, it'll help everyone.

Reply:
> > I no longer have a warning when switching
> 
> You never mentioned any warnings: If you post the warnings you get, it'll help everyone.

I am using CMake 3.21, I cannot get the warning again, i hadn't paid more attention than that, It is perhaps another of my modifications which caused this warning, it is no longer there by reverting to version 2.8.12, I go back to version 2.8.12 ?

Reply:
Tried myself, `target_sources` is an issue. I'm OK with bumping min version to 3.0.